commands/doit.go

```diff
@@ -93,7 +103,26 @@ func initConfig() {
 	}
 
 	viper.SetDefault("output", "text")
+}
+
+func findConfig() (string, error) {
+	if cfgFile != "" {
+		return cfgFile, nil
+	}
+
+	legacyConfigPath := filepath.Join(homeDir(), ".doctlcfg")
```

Comment:
since this is in `.config/doctl`, any reason not to just call this `config.json`?

```
$ tree ~/.config
/Users/antoine/.config
├── fish
     └── config.fish
└── gocode
     └── config.json
...
```